styles/all/template/event/forumlist_body_category_header_before.html

```diff
@@ -0,0 +1,7 @@
+<!-- IF forumrow.S_IS_CAT -->
```

Comment:
If you are in viewforum.php (with subforums), there might not be a category for some of these forums, but you could still want to hide the subforums of the particular forum.

Example: https://www.phpbb.com/community/viewforum.php?f=446

So you might want to be able to collapse "uncategorized" forums as well using `<!-- IF forumrow.S_IS_CAT or forumrow.S_FIRST_ROW or forumrow.S_NO_CAT  -->`

Something to think about.